### PR TITLE
Scale starter PvP shield with highest tech on map + bunch of small stuff

### PR DESCRIPTION
**Pull request by blubFisch**

### CODE REVIEW

File: maps/scrap_towny_ffa/evolution.lua [hunk 1]
```diff
@@ -483,6 +484,16 @@ local function get_relative_worm_evolution(position)
...
 end
 
+function Public.get_highest_evolution()
+    local this = ScenarioTable.get_table()
+    local relative_evolution = 0.0
+    local max_evo = 0
+    for _, town_center in pairs(this.town_centers) do
+        max_evo = math_max(town_center.evolution.worms, max_evo)
+    end
+    return relative_evolution
```
Comment (hanakocz) [inline maps/scrap_towny_ffa/evolution.lua]: This always returns 0, maybe you mean to return max_evo ?

Comment (blubFisch) [reply maps/scrap_towny_ffa/evolution.lua]: Ah good catch, lemme fix

Comment (blubFisch) [reply maps/scrap_towny_ffa/evolution.lua]: I think this was working okay, but I put in a more clear version anyways

**Comment by blubFisch** [comment]
Also, can u merge this squashed please? To keep history clean :-)

File: maps/scrap_towny_ffa/evolution.lua (hunk 1 above)
Comment (blubFisch) [reply maps/scrap_towny_ffa/evolution.lua]: Fix v2 is up... for some reason my IDE's code model fails on this file

**Pull request merged**